docs/source/migration.mdx

```diff
@@ -72,68 +202,60 @@ Apollo Server 4 removes the below integration packages:
+But wait: there's more! The following packages have *also* been combined into `@apollo/server`:
+- [`apollo-server-errors`](https://www.npmjs.com/package/apollo-server-errors)
+- [`apollo-server-plugin-base`](https://www.npmjs.com/package/apollo-server-plugin-base)
+- [`apollo-server-types`](https://www.npmjs.com/package/apollo-server-types)
 
-If you want to use either `cors` or `body-parser`, you can manually add both libraries to your framework-specific middleware.
 
-Below is an example of setting up `cors` and `body-parser` using the [`expressMiddleware`](#expressmiddleware) function:
+### Other renamed packages
 
-<MultiCodeBlock>
+The `apollo-server-plugin-response-cache` and `apollo-server-plugin-operation-registry` plugin packages have been renamed to `@apollo/server-plugin-response-cache` and `@apollo/server-plugin-operation-registry` respectively.
 
-```ts
-import { ApolloServer, ApolloServerPluginDrainHttpServer } from '@apollo/server';
-import { expressMiddleware } from '@apollo/server/express4';
-import express from 'express';
-import http from 'http';
-import cors from 'cors';
-import bodyParser from 'body-parser';
+The [`apollo-reporting-protobuf`](https://www.npmjs.com/package/apollo-reporting-protobuf) (which is an internal implementation detail of the usage reporting plugin) has been renamed to `@apollo/usage-reporting-protobuf`.
 
-// Set up Express
-const app = express();
-const httpServer = http.createServer(app);
+Note that once AS4 is released, all actively maintained Apollo packages will start with `@apollo/`. This leaves the `apollo-` namespace open for community integration packages (e.g., `apollo-server-fastify`).
 
-// Set up Apollo Server
-const server = new ApolloServer({
-  typeDefs,
-  resolvers,
-  plugins: [ApolloServerPluginDrainHttpServer({ httpServer })],
-});
-await server.start();
 
-// We set up our Express server's middleware:
-// 1. using the `cors` and `body-parser` libraries, and
-// 2. passing in an `ApolloServer` instance to the `expressMiddleware` function,
-// attaching Apollo Server to our Express server
-app.use(cors(), bodyParser.json(), expressMiddleware(server));
-await new Promise<void>((resolve) => httpServer.listen({ port: 4000 }, resolve));
+## Bumped dependencies
+
+### Node.js
 
-console.log(`🚀 Server ready at http://localhost:4000`);
-```
+Apollo Server 4 supports Node.js 14 and later. (Apollo Server 3 supports Node.js 12.) This includes all [LTS and Current versions at the time of release](https://nodejs.org/en/about/releases/).
 
-</MultiCodeBlock>
+If you're using Node.js 12, upgrade your runtime before upgrading to Apollo Server 4.
+
+### `graphql`
+
+Apollo Server has a peer dependency on [`graphql`](https://www.npmjs.com/package/graphql) (the core JS GraphQL implementation). Apollo Server 4 supports `graphql` v16.3.0 and later. (Apollo Server 3 supported `graphql` v15.3.0 through v16.)
```

Comment:
Wonder if we can expand to all of v16, I forget if we actually make use of a new feature in any of the v16 minor releases?

Reply:
I think we might, although there's something to be said for just requiring the latest whenever we do a major bump so we don't later wish we had a slightly newer minimum.

Reply:
I'm going to assume we're sticking with this requirement

Reply:
(or go to 16.5 :) )

Reply:
actually it already is 16.5! I think for some GraphQLError stuff